streams/hash_object.proto

```diff
@@ -0,0 +1,55 @@
+syntax = "proto3";
+
+package proto;
+
+/*-
+ * ‌
+ * Hedera Network Services Protobuf
+ * ​
+ * Copyright (C) 2018 - 2022 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+option java_package = "com.hederahashgraph.api.proto.java";
+option java_multiple_files = true;
+
+/**
+ * List of hash algorithms
+ */
+enum HashAlgorithm {
+  HASH_UNKNOWN = 0;
+  SHA_384 = 1;
+}
+
+/**
+ * A Hash Object consisting of algorithm used, hash length and the hash bytes
```

Comment:
Listing the fields is not very useful and would require changes later to keep it in sync. Instead we should explain why a wrapper object is useful.

```suggestion
 * Encapsulates an object hash so that additional hash algorithms can be added in the future without requiring a breaking change.
```